src/containers/post/post.component.jsx

```diff
@@ -10,20 +17,36 @@ const Post = (props) => {
         <button
           className="bg-gray-300 hover:bg-gray-400 w-20 h-10 rounded-l-lg border-r-2 border-gray-600"
           onClick={() =>
-            voteForOption({ optionName: "option_a", postId: props.id })
+            handleVote({ optionName: "option_a", postId: props.id })
           }
         >
           {props.optionAName}
         </button>
         <button
           className="bg-gray-300 hover:bg-gray-400 w-20 h-10 rounded-r-lg "
           onClick={() =>
-            voteForOption({ optionName: "option_b", postId: props.id })
+            handleVote({ optionName: "option_b", postId: props.id })
           }
         >
           {props.optionBName}
         </button>
       </div>
+      <div className="flex justify-center">
+        <div
+          className={`${
+            voteView ? "show" : "hidden"
+          } bg-gray-300 hover:bg-gray-400 w-20 h-10 rounded-l-lg border-r-2 border-gray-600`}
+        >
+          {props.optionAName}: {props.optionA + 1}
+        </div>
+        <div
+          className={`${
+            voteView ? "show" : "hidden"
+          } bg-gray-300 hover:bg-gray-400 w-20 h-10 rounded-r-lg`}
+        >
+          {props.optionBName}: {props.optionB + 1}
+        </div>
+      </div>
```

Comment:
Not going to block this for now, but this is an example of repeated code that can turned into a function such as `renderVoteOption` and takes in a couple of arguments and decides whether to render optionA info or optionB info

Reply:
We want both of them rendering though

Reply:
Yeah can render both by calling the function twice with different parameters